solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/RejoinLeaderElectionPayload.java

```diff
@@ -26,13 +26,9 @@ public class RejoinLeaderElectionPayload implements ReflectMapWriter {
   // ones are meant to be required without that being specified on the v1 API or elsewhere
   @JsonProperty public String collection;
 
-  @JsonProperty public String shard;
```

Comment:
Is this backwards-compatible? I'm not super familiar with this, but from what I can see, there are some inputs that are no longer needed.  It seems they weren't really used before (yes?).  What would happen if a client supplies it?

CC @gerlowskija you might know if this produces a failure or not

Reply:
Thanks for pointing this @dsmiley.
I did a test locally and it seems to me any additional property in the Json payload here is simply ignored. Since this change is only removing some properties, I don't think we can have backwards-compatibility issue.